apps/liquidity-provision-dashboard/src/app/components/detail/providers/providers.tsx

```diff
@@ -0,0 +1,77 @@
+import { useCallback } from 'react';
+import { AgGridColumn } from 'ag-grid-react';
+
+import type { GetRowIdParams } from 'ag-grid-community';
+import { t } from '@vegaprotocol/react-helpers';
+
+import type { LiquidityProvisionFieldsFragment } from '@vegaprotocol/liquidity';
+import { formatWithAsset } from '@vegaprotocol/liquidity';
+
+import { Grid } from '../../grid';
+
+const formatToHours = ({ value }: { value?: string | null }) => {
+  if (!value) {
+    return '-';
+  }
+
+  const MS_IN_HOUR = 1000 * 60 * 60;
+  const created = new Date(value).getTime();
+  const now = new Date().getTime();
+  return `${Math.round(Math.abs(now - created) / MS_IN_HOUR)}h`;
+};
+
+export const Providers = ({
```

Comment:
This isn't a structural question really - but are we worried about potential confusion between the term `Provider` between Liquidity Providers and React Context Providers?

('No' is a reasonable answer of course!)

Reply:
Haha that is a fair question (we also use `dataProviders` for fetching data 😅 ). Do you think I should rename it to LiquidityProviders? Or maybe `Orders`... What do you think? :)

Reply:
Not sure! Is there much precedent for stuff like this in the project? It's easy enough for us to change in the future anyway, so I don't imagine holding up on this.

Reply:
I'm not too worried about the confusion around this, though perhaps a slightly more descriptive name might be suitable? Perhaps LPProvidersGrid?